src/Components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import '../../styles/SearchBar.css';
+import Container from '@material-ui/core/Container';
+
+export class SearchBar extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    return (
+      <Container maxWidth="sm" id="main-view">
+        <form id="search-form" noValidate autoComplete="off">
```

Comment:
Nice job on implementing the search bar. It _might_ be helpful (from a usability perspective) to add a 'Go' or 'Submit' button? The user may not assume to hit Enter :)